src/main/java/project/controller/LoginController.java

```diff
@@ -64,27 +70,19 @@ public LoginController(UserService userService, MockDataService mockDataService)
             // user doesn't exist and is a new user
             // user is created in database and we return 201
 
-            String accessTokenGeneratedFromServer = userService.createUser(id, name, email);
+            String accessTokenGeneratedFromServer = userService.createUser(id, name, email, firstName, pictureUrl);
             if (accessTokenGeneratedFromServer == null) return new ResponseEntity<LoginResponse>(HttpStatus.SERVICE_UNAVAILABLE);
-            user = new User(accessTokenGeneratedFromServer, id, name, email, 0.0, 0, ""); // TODO: add profile picture url
+            user = new User(accessTokenGeneratedFromServer, id, name, email, 0.0, 0, pictureUrl, firstName,
+                    null); // phone number is null, as we can't get it from facebook
 
-            LoginResponse loginResponse = new LoginResponse(
-                    mockDataService.getMockDriverList(),
-                    mockDataService.getMockRiderList(),
-                    user);
+            LoginResponse loginResponse = new LoginResponse(user);
```

Comment:
only returning the user for `/login` endpoint, as the client is not using the driver/rider reponse